examples/powercli/nsx-t_tag-sync/stack.yml

```diff
@@ -0,0 +1,16 @@
+provider:
+  name: openfaas
+  gateway: https://phxlvveba01.itplab.local
+functions:
+  nsxttagsync:
+    lang: powercli
+    handler: ./handler
+    image: cmstraka/veba-powercli-tag-sync-to-nsxt:latest
```

Comment:
Please name this `vmware/veba-powercli-tag-sync-to-nsxt:latest` since we'll build this container and host it upon approval

Reply:
Done!